cmd/exo/cmd/sos_remove.go

```diff
@@ -82,7 +82,7 @@ var removeCmd = &cobra.Command{
 					nbFile++
 				}
 				if nbFile == 0 {
-					fmt.Fprintf(os.Stderr, "rm: cannot remove '%s': No such file or directory\n", arg)
+					fmt.Fprintf(os.Stderr, "rm: cannot remove '%s': No such file or directory\n", arg) // nolint: errcheck
```

Comment:
this doesn't return a proper error code

Reply:
https://github.com/exoscale/egoscale/pull/182/commits/17cbbb05a50b3a58113dcd842ff39114ef873803